index.html

```diff
@@ -956,7 +926,771 @@ <h4>Proof Verification (experimental-mldsa44-2024)</h4>
+proof options (|options|). A <em>verification result</em>
+represented as a boolean value is produced as output.
+          </p>
+
+          <ol class="algorithm">
+            <li>
+Let |publicKeyBytes| be the result of retrieving the
+public key bytes associated with the
+|options|.|verificationMethod| value as described in the
+Data Integrity [[VC-DATA-INTEGRITY]] specification,
+<a data-cite="vc-data-integrity#algorithms">
+Section 4: Retrieve Verification Method</a>.
+            </li>
+            <li>
+Let |verificationResult| be the result of applying the verification
+algorithm from the Falcon Digital Signature scheme [[FALCON]],
+with |hashData| as the data to be verified against the
+|proofBytes| using the public key specified by
+|publicKeyBytes|.
+            </li>
+            <li>
+Return |verificationResult| as the <em>verification result</em>.
+            </li>
+          </ol>
+
+        </section>
+      </section>
+    
+
+      <section>
+        <h3>experimental-sqi-2025</h3>
+
+        <p>
+The `experimental-sqi-2025` cryptographic suite takes an
+input document, canonicalizes the document using the Universal RDF Dataset
+Canonicalization Algorithm [[RDF-CANON]], and then cryptographically hashes and
+signs the output resulting in the production of a data integrity proof. The
+algorithms in this section also include the verification of such a data
+integrity proof.
+        </p>
+
+        <section>
+          <h4>Create Proof (experimental-sqi-2025)</h4>
+
+          <p>
+The following algorithm specifies how to create a [=data integrity proof=] given
+an <a>unsecured data document</a>. Required inputs are an
+<a>unsecured data document</a> ([=map=] |unsecuredDocument|), and a set of proof
+options ([=map=] |options|). A [=data integrity proof=] ([=map=]), or an error,
+is produced as output.
+          </p>
+
+          <ol class="algorithm">
+            <li>
+Let |proof| be a clone of the proof options, |options|.
+            </li>
+            <li>
+Let |proofConfig| be the result of running the algorithm in
+Section [[[#proof-configuration]]] with
+|options| and `experimental-sqi-2025` passed as parameters.
+            </li>
+            <li>
+Let |transformedData| be the result of running the algorithm in Section <a
+href="#transformation"></a> with |unsecuredDocument|, |options| and 
+`experimental-sqi-2025` passed as parameters.
+            </li>
+            <li>
+Let |hashData| be the result of running the algorithm in Section
+[[[#hashing]]] with |transformedData| and |proofConfig|
+passed as a parameters.
+            </li>
+            <li>
+Let |proofBytes| be the result of running the algorithm in Section
+[[[#proof-serialization-experimental-sqi-2025]]] with |hashData| and
+|options| passed as parameters.
+            </li>
+            <li>
+Let |proof|.|proofValue| be a <a data-cite="VC-DATA-INTEGRITY#multibase-0">
+base58-btc-encoded Multibase value</a> of the |proofBytes|.
```

Comment:
We'll need to adjust these encodings. But, again, after de-duplication we can just do this once, just noting.

Reply:
How do you mean @dlongley?

Is there additional deduplication you would like to see in this PR?